tests/furusiya/model/entities/test_npc.py

```diff
@@ -0,0 +1,47 @@
+from unittest.mock import Mock
+
+import pytest
+
+from model.components.ai.base import AI
+from model.entities.npc import NPC
+
+
+class AITest(AI):
+    def __init__(self, owner=Mock()):
+        super().__init__(owner)
+
+
+class AITest1(AITest):
```

Comment:
This would be clearer if you called it, like `StunnedAI` or `OverlordAI` or something, but it's fine as is.